project/npda/views/submissions.py

```diff
@@ -377,6 +379,13 @@ async def upload_csv(request):
             ip_address=get_client_ip(request),
         )
 
+        # Gather unique patient and visit counts and update the submission
+        patient_count, visit_per_patient_count, total_rows = gather_unique_patient_and_visit_counts(parsed_csv.df, is_jersey)
```

Comment:
Could we do this in the `create_csv_submission` helper? That way the fields will be set when using the `upload_csv` management command too